src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1023,19 +1015,17 @@ public function testRunWithDispatcherAddingInputOptions()
         $this->assertEquals('some test value', $extraValue);
     }
 
+    /**
+     * @group legacy
+     */
     public function testTerminalDimensions()
     {
         $application = new Application();
+        $application->setTerminalDimensions(120, 80);
```

Comment:
the fact that you move this line in the test is hiding the BC break I catched above, as the test was precisely covering the case of changing dimensions after reading them.